src/circuit.rs

```diff
@@ -37,24 +37,6 @@ pub struct Cell {
     column: Column<Any>,
 }
 
-/// A permutation configured by a chip.
-#[derive(Clone, Debug)]
-pub struct Permutation {
```

Comment:
I'm not sure that we want to move `Permutation` into `halo2::plonk::circuit`. Per #165 (where we want to get rid of `halo2::plonk::circuit`), the goal is for `halo2::circuit` to contain the types that most crate users (circuit implementors) deal with, while `halo2::plonk` contains the inner details that few people will use directly beyond `create_proof` and `verify_proof`.

Reply:
Hmm, I now see that @ebfull suggested the move. I'm only weakly against it, so we can keep it as-is and then maybe move it back during #165.